src/components/RepeatWords/RepeatWords.tsx

```diff
@@ -0,0 +1,137 @@
+import * as S from './StyleRepeatWords';
+import ButtonIcon from 'components/Shared/Buttons/ButtonIcon';
+import { TitleBig } from 'components/Shared/Atoms/Title';
+import { inputNameElement } from 'helpers/mixins';
+import fetchWithToken from 'API/api';
+import { useState, useEffect } from 'react';
+
+interface Word {
+  _id: string;
+  userId: string;
+  basicWord: string;
+  transWord: string;
+  addLang: number;
+  status: number;
+  createdDate: string;
+  updatedDate: string;
+  daysRepeat: number;
+  __v: number;
+}
+
+interface RepeatWordsProps {
+  daysRepeat: number;
+}
+
+const RepeatWords: React.FC<RepeatWordsProps> = ({ daysRepeat }) => {
+  const [wordsRepeat, setWordsRepeat] = useState<Word[]>([]);
+  const [currentWordIndex, setCurrentWordIndex] = useState<number>(0);
```

Comment:
tu nie musisz robic `<number>`, bo jeśli value jest number to ts sam o tym wie. 
wyżej jest to ptrzebne bo samo `[]` nic mus nie mówi
tam samo niżej `<string>` nie jest potrzebne. on wie, ze jak jest `''` to będzie typ string